### ISSUE TITLE
[WEBUI] TypeError: r.files is undefined

### ISSUE THREAD

**Issue by skibbipl** [autobrr/autobrr#1380]
**Version**
1.35.1

**Describe the bug**
```
TypeError: r.files is undefined

at lC  screens/Logs.tsx:197  {data && data.files.length > 0 ? (
```

**To Reproduce**
1. Go to 'logs'
2. See error

**Expected behavior**
Logs displayed.

Downgrading to 1.35 fixes the issue.

**Comment by grublets** [comment]
Came here to add the same issue.

**Comment by zze0s** [comment]
Dupe of #1375

### This is a frontend/webui/cookie issues.

Try another browser or incognito window. If it works there then clear your cookies.

I won't close this for a while.

**Comment by RsX88** [comment]
Same here. 
Cleared cookies and everything is working again.
Thanks

**Comment by grublets** [comment]
Yep, all good now. Minor panic averted!

**Comment by martylukyy** [comment]
Should be fixed by #1338 
Feel free to comment or reopen in case the PR mentioned above did not resolve this issue.
